arangod/V8Server/v8-views.cpp

```diff
@@ -319,7 +332,12 @@ static void JS_DropViewVocbaseObj(
     TRI_V8_THROW_EXCEPTION_MESSAGE(TRI_ERROR_FORBIDDEN, "insufficient rights to drop view");
   }
 
-  auto res = view->vocbase().dropView(view->id(), allowDropSystem);
+  // prevent dropping of system views
+  if (!allowDropSystem && view->system()) {
```

Comment:
is there such a thing as system views ?

Reply:
apparently, since this is a condition from existing code